### PR TITLE
use constants for ball throw count

### PR DESCRIPTION
**Pull request by PidgeyALT**
henlo am neyou to dis but i want help you

### CODE REVIEW

File: include/battle.h [hunk 1]
```diff
@@ -12,6 +12,8 @@
 #include "battle_util2.h"
 #include "battle_bg.h"
 
+#include "pokeball.h" // i'm not sure about this one
```
Comment (PidgeyALT) [inline include/battle.h:15]: i'm unsure about including this header here does anyone have anything to say about this?

**Comment by PikalaxALT** [comment]
Review note: Pull request template was ignored for this PR. Requester's discord name is MCboy#0891

File: include/battle.h (hunk 1 above)
Comment (GriffinRichards) [reply include/battle.h:15]: It's fine, but group it with the other headers and remove the comment

File: include/battle.h [hunk 2]
```diff
@@ -255,7 +257,7 @@ struct BattleResults
     u16 caughtMonSpecies;     // 0x28
     u8 caughtMonNick[POKEMON_NAME_LENGTH + 1];     // 0x2A
     u8 filler35;           // 0x35
-    u8 catchAttempts[11];     // 0x36
+    u8 catchAttempts[POKEBALL_COUNT-1];     // 0x36 Doesn't include Master ball
```
Comment (GriffinRichards) [inline include/battle.h:260]: Space around `-`, and throughout PR

**Review by GriffinRichards — CHANGES_REQUESTED** [review]

**Review by GriffinRichards — APPROVED** [review]

**Pull request merged**